projects/vertical-components/assets/vertical-theme.scss

```diff
@@ -0,0 +1,66 @@
+@import '~@angular/material/theming';
+@import '../src/lib/button/button.scss';
```

Comment:
Würde ich hier nicht machen. Wenn ich mich jetzt dazu entscheide ein anderes Theme zu laden. Sagen wir an PackEx-Theme, dann müsste ich ja trotzdem das hier importieren um die Funktionen für den Button zu erhalten. Das hier könnten wir in einer Datei vertical-components-core unterbringen. Sorry wenn wir da vorhin aneinander vorbei gesprochen haben